Formula/s/slepc.rb

```diff
@@ -0,0 +1,46 @@
+class Slepc < Formula
+  desc "Scalable Library for Eigenvalue Problem Computations (real)"
+  homepage "https://slepc.upv.es"
+  url "https://slepc.upv.es/download/distrib/slepc-3.22.2.tar.gz"
+  sha256 "b60e58b2fa5eb7db05ce5e3a585811b43b1cc7cf89c32266e37b05f0cefd8899"
+  license "BSD-2-Clause"
+
+  livecheck do
+    url "https://slepc.upv.es/download/distrib/"
+    regex(/href=.*?slepc[._-]v?(\d+(?:\.\d+)+)\.t/i)
+  end
+
+  depends_on "open-mpi"
+  depends_on "openblas"
+  depends_on "petsc"
+
+  uses_from_macos "python" => :build
```

Comment:
```suggestion
  uses_from_macos "python" => :build

  on_macos do
    depends_on "gcc"
  end
```
Since you have indirect linkage to it.